client/extensions/woocommerce/woocommerce-services/views/live-rates-carriers-list/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React from 'react'
+import { localize } from 'i18n-calypso'
+
+/**
+ * Internal dependencies
+ */
+import ExtendedHeader from 'woocommerce/components/extended-header'
+import CarriersList from './carriers-list'
+
+const supportedCarrierIds = ['wc_services_usps', 'wc_services_dhlexpress'];
```

Comment:
When testing against staging, there's also FedEx - need to whitelist the carriers to generate less possible confusion when other people are testing against the staging server

Reply:
:information_source: Missing spaces inside brackets.